sqld/src/rpc/mod.rs

```diff
@@ -21,14 +23,15 @@ pub async fn run_rpc_server<F>(
     ca_cert_path: Option<PathBuf>,
     factory: F,
     logger: Arc<WalLogger>,
+    hello_response: HelloResponse,
```

Comment:
I think that this information should be retrieved from the `WalLogger`

Reply:
done.